sql/src/test/java/org/opensearch/sql/sql/antlr/HighlightTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql.antlr;
+
+import org.junit.jupiter.api.Test;
+
+public class HighlightTest extends SQLParserTest {
+  @Test
+  void single_field_test() {
+    acceptQuery("SELECT HIGHLIGHT(Tags) FROM Index WHERE MATCH(Tags, 'Time')");
+  }
+
+  @Test
+  void multiple_highlights_test() {
+    acceptQuery("SELECT HIGHLIGHT(Tags), HIGHLIGHT(Body) FROM Index "
+        + "WHERE MULTI_MATCH([Tags, Body], 'Time')");
+  }
+
+  @Test
+  void wildcard_test() {
+    acceptQuery("SELECT HIGHLIGHT('T*') FROM Index "
+        + "WHERE MULTI_MATCH([Tags, Body], 'Time')");
+  }
+
+  @Test
+  void highlight_all_test() {
+    acceptQuery("SELECT HIGHLIGHT('*') FROM Index WHERE MULTI_MATCH([Tags, Body], 'Time')");
+  }
```

Comment:
We should maybe add negative tests here too, to avoid adding Highlight to the WHERE, FROM, or ORDER BY clauses?

Reply:
I don't think fixing this is in the scope of highlight. Here is a list of types of functions that can be used in a WHERE clause that will still pass parsing: [OpenSearchSQLParser.g4#L296-L302](https://github.com/opensearch-project/sql/blob/main/sql/src/main/antlr/OpenSearchSQLParser.g4#L296-L302)